crates/sui-storage/src/indexes.rs

```diff
@@ -233,6 +232,7 @@ pub struct IndexStore {
     caches: IndexStoreCaches,
     metrics: Arc<IndexStoreMetrics>,
     max_type_length: u64,
+    remove_deprecated_tables: bool,
```

Comment:
Do we have a way to unconditionally remove `loaded_child_object_versions` and conditionally remove the other two? That way everyone can benefit from that one being nuked while people can opt into the other two being removed?

Reply:
no, we don't at the moment. We can create a config with explicit list of column families to be removed. Or nuke loaded_child_object_versions unconditionally.
Ofc subject for discussion, but I kind of like it this way. Puts some minor pressure on FN operator to start migration process from JSON RPC

Reply:
Ok i'm fine with leaving it as-is (single config to remove all deprecated cfs) so long as we can unconditionally nuke `loaded_child_object_versions` for everyone regardless of this flag since there's no reason to keep it.